services/downloads.js

```diff
@@ -27,6 +29,7 @@ function renderDownloadsBadge({
   labelOverride,
   colorOverride,
   messageSuffixOverride,
+  versionedLabelPrefix = 'downloads',
```

Comment:
Added to support use cases where classes want to support tag/version specific download counts but are using a label other than `downloads`